fl4health/clients/basic_client.py

```diff
@@ -53,15 +50,8 @@ def __init__(
         self.train_loss_meter = LossMeter.get_meter_by_type(loss_meter_type)
         self.val_loss_meter = LossMeter.get_meter_by_type(loss_meter_type)
 
-        # Define mapping from prediction key to meter to pass to MetricMeterManager constructor for train and val
-        train_key_to_meter_map = {
-            "prediction": MetricMeter.get_meter_by_type(self.metrics, metric_meter_type, "train_meter")
-        }
-        self.train_metric_meter_mngr = MetricMeterManager(train_key_to_meter_map)
-        val_key_to_meter_map = {
-            "prediction": MetricMeter.get_meter_by_type(self.metrics, metric_meter_type, "val_meter")
-        }
-        self.val_metric_meter_mngr = MetricMeterManager(val_key_to_meter_map)
+        self.train_metric_meter_mngr = MetricManager(metrics=self.metrics, metric_mngr_name="train")
```

Comment:
Since these are now "MetricManager" objects, should we change the names from `train_metric_meter_mngr` to `train_metric_manager ` and `val_metric_meter_mngr ` to `val_metric_manager` throughout?